fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/InPredicateOperator.java

```diff
@@ -10,20 +10,34 @@
 
 public class InPredicateOperator extends PredicateOperator {
     private final boolean isNotIn;
+    private final boolean isSubquery;
```

Comment:
Do we need to change the equal and hash code methods?

Reply:
Yes. Updated